static/js/index.js

```diff
@@ -0,0 +1,127 @@
+const API_PREFIX = 'api';
+const baseUrl = window.location.origin;
+
+const setScrapingLogs = (logs) => {
+  const logsNode = document.getElementById('scraping-logs');
+  const root = document.createElement('div');
+  logs.forEach((el) => {
+    const pNode = document.createElement('p');
+    pNode.innerText = el;
+    root.appendChild(pNode);
+  });
+  logsNode.appendChild(root)
+};
+
+const setRadioButtons = (scraperSettings) => {
+  document.querySelector(`[name="loop-radio-group"][value="${scraperSettings.loop_mode.toString()}"]`).checked = true;
+  document.querySelector(`input[name="media-radio-group"][value="${scraperSettings.media_mode}"]`).checked = true;
+  document.querySelector(`input[name="ids-radio-group"][value="${scraperSettings.ids_source}"]`).checked = true;
+};
+
+const checkShutdown = async () => {
+  const makeStatusRequest = async () => {
+    const requestUrl = `${baseUrl}/${API_PREFIX}/scraper/status/`;
+    return await (await fetch(requestUrl)).json();
+  };
+
+  const CHECK_STATUS_EVERY_MS = 2000;
+  const updatePage = async () => {
+  const response = await makeStatusRequest();
+  if (response.status === 'stopped') {
+    location.reload();
+  } else {
+    setTimeout(updatePage, CHECK_STATUS_EVERY_MS);
+  }
+  };
+  setTimeout(updatePage, CHECK_STATUS_EVERY_MS);
+};
+
+const updateCommandButton = (buttonStatus) => {
+  const buttonNode = document.getElementById('button-container');
+  const commandButton = document.createElement('button')
+  commandButton.id = 'command-button'
+  const isRunning = buttonStatus == 'running'
+  const isShuttingDown = buttonStatus == 'shutdown'
+  commandButton.disabled = isShuttingDown ? true : false;
+  commandButton.classList = isRunning ? 'btn stop-btn' : isShuttingDown ? 'btn update-btn' : 'btn start-btn';
+  commandButton.innerText = isRunning ? 'Stop' : isShuttingDown ? 'updating...' : 'Start';
+  commandButton.value = isRunning ? 'stop' : 'start';
```

Comment:
Use a template literals for all of this with a standard value.

Then you can do 

if (isRunning) {
  commandButton.classlist.add(stop-btn);
} etc...

Reply:
Done